content/docs/start/experiments.md

```diff
@@ -2,293 +2,225 @@
+run the commands in this document after cloning the repository and installing
+the requirements.
 
-```git
- featurize:
--  max_features: 1500
-+  max_features: 3000
-```
-
-Any edits to <abbr>dependencies</abbr> (parameters or source code) will be
-reflected in the experiment run.
+#### Clone the project and create virtual environment
 
-</details>
+Please clone the project and create a virtual environment.
 
-`dvc exp diff` compares experiments:
+> We strongly recommend to create a virtual environment to keep the libraries we
+> use isolated from the rest of your system. This prevents version conflicts.
 
 ```dvc
-$ dvc exp diff
-Path         Metric    Value    Change
-scores.json  avg_prec  0.56191  0.009322
-scores.json  roc_auc   0.93345  0.018087
-
-Path         Param                   Value    Change
-params.yaml  featurize.max_features  3000     1500
+$ git clone https://github.com/iterative/get-started-experiments -b get-started
+$ cd get-started-experiments
+$ virtualenv .venv
+$ . .venv/bin/activate
+$ python -m pip install -r requirements.txt
 ```
 
-## Queueing experiments
+#### Get the data set
 
-So far, we have been tuning the `featurize` stage, but there are also parameters
-for the `train` stage (which trains a
-[random forest classifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html)).
-
-These are the `train` parameters from `params.yaml`:
-
-```yaml
-train:
-  seed: 20170428
-  n_est: 50
-  min_split: 2
-```
-
-Let's set up experiments with different hyperparameters. We can use the
-`--queue` flag to define all the combinations we want to try without executing
-anything (yet):
+The repository we cloned doesn't contain the dataset. Instead of storing the
+data in Git repository, we use DVC to retrieve from a shared data store. In this
+case, we use `dvc pull` to update the missing data files.
 
 ```dvc
-$ dvc exp run --queue -S train.min_split=8
-Queued experiment 'd3f6d1e' for future execution.
-$ dvc exp run --queue -S train.min_split=64
-Queued experiment 'f1810e0' for future execution.
-$ dvc exp run --queue -S train.min_split=2 -S train.n_est=100
-Queued experiment '7323ea2' for future execution.
-$ dvc exp run --queue -S train.min_split=8 -S train.n_est=100
-Queued experiment 'c605382' for future execution.
-$ dvc exp run --queue -S train.min_split=64 -S train.n_est=100
-Queued experiment '0cdee86' for future execution.
+$ dvc pull
 ```
 
-Next, run all (`--run-all`) queued experiments in parallel (using `--jobs`):
+The repository already contains the necessary configuration to run the
+experiments.
 
-```dvc
-$ dvc exp run --run-all --jobs 2
-```
+</details>
 
-## Comparing many experiments
+## Running the experiment with default parameters
 
-To compare all of these experiments, we need more than `diff`. `dvc exp show`
-compares any number of experiments in one table:
+The purpose of `dvc exp` subcommands is to run the experiments without
```

Comment:
without committing something to Git (here and in other places) - people who will be reading this don't know why would they need to commit anything into Git, they won't know about metrics and params.

I think there should be some general language initially:

`the purpose of the commands ... is to let you run, capture, compare one or multiple experiments at once as you iterate on your code, data, hyperparameters`

also `The purpose of `dvc exp` subcommands is to run the experiments` is not correct - there are other commands besides `run`

Reply:
I revised this.